webcap/browser.py

```diff
@@ -197,7 +198,11 @@ async def request(self, command, sessionId=None, retry=False, **params):
                 if sessionId:
                     request["sessionId"] = sessionId
                 await self._send_request(request)
-                response = await future
```

Comment:
Ran across this bug in master causing screenshots to hang indefinitely only to find it here in the dev branch :) You should merge it. Also, great tool!